README.md

```diff
@@ -1,15 +1,203 @@
-# An alternative http library and web framework inspired by hyper
+<h1 align="center">Xitca-Web</h1>
+<p align="center">
+  <b align="center"><a href="README.md">Readme</a></b> |
+  <b><a href="https://github.com/HFQR/xitca-web">GitHub</a></b> |
+  <b><a href="https://docs.rs/xitca-web/latest/xitca_web/">Documentation</a></b>
+  <br /><br />
+  <a href="#">
+    <img
+      alt="GitHub code size in bytes"
+      src="https://img.shields.io/github/languages/code-size/HFQR/xitca-web?style=flat-square"
+    />
+  </a>
+  <a href=""
+    ><img
+      alt="Maintenance"
+      src="https://img.shields.io/maintenance/yes/2024?style=flat-square"
+    />
+  </a>
+  <br />
+  <br />
+  <i>
+    A HTTP library and web framework written in 100% safe Rust with zero-copy
+    serialization/deserilization support
+  </i>
+</p>
 
-## Minimum Supported Rust Version
-- 1.79 [^1]
+<br />
+<img src="./images/server.jpg" alt="An image of a server" align="center" />
+<br />
 
-## Motivation
-- Less synchronization and thread per core design is used.
-- 100% safe Rust.[^2]
-- Low memory footprint. Avoid (pre)allocation when possible.
-- Lightweight dependency tree. Avoid adding unnecessary import when possible. Prefer no proc macro code generation when possible(proc macro feature are still offered as opt-in instead of opt-out).
-- Compact and simple code base. Reduce the barrier of understanding of source code for easier contributing.
-- Good interop with other crates. `tokio`(for async runtime) and `http`(for http types) are used directly as dependency.
+<details>
+  <summary><b>Table of Contents</b></summary>
+  <p>
 
-[^1]: `http-file` excluded due to design issue.
-[^2]: only guaranteed inside project's own code. unsafe can and is used by dependencies
+  - [🚀 Features](#features-)
+  - [📊 Minimum Supported Rust Version](#minimum-supported-rust-version-)
+  - [⚡ Quick Start](#quick-start-)
+  - [🗨️ FAQ (Frequently Asked Questions)](#faq-frequently-asked-questions-%EF%B8%8F)
+  - [📣 More Contributors Wanted](#more-contributors-wanted-)
+  - [💖 Supporting Xitca-web](#supporting-xitca-web-)
+  - [📘 Documentation](#documentation-)
+  - [🛣️ Roadmap](#roadmap-%EF%B8%8F)
+  - [🙋 Contributing](#contributing-)
+  - [📜 License](#license-)
+  - [🤝 Credits](#credits-)
+
+  </p>
+</details>
+
+# Features 🚀
+
+- 🎗️ Supports HTTP/1.x and HTTP/2
```

Comment:
`xitca-web` project also supports HTTP/3 (with `xitca-http` specifically)

Reply:
> `xitca-web` project also supports HTTP/3 (with `xitca-http` specifically)

Ok sure, I will mention about this also in the `features` section. :slightly_smiling_face: